.github/workflows/dart.yml

```diff
@@ -0,0 +1,45 @@
+name: Dart
+
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: windows-latest
+    environment: WindowsEnv

```

Comment:
What does this do?

Reply:
This specifies an environment containing the secrets used by the workflow.

Reply:
We should make this example as simple as possible. Can we cut this if we use repository secrets instead?